### PR TITLE
Data Server Updates

### PR DESCRIPTION
**Pull request by thpang**

### CODE REVIEW

**Comment by NormJohnIV** [comment]
LGTM

File: docs/CONFIG-VARS.md [hunk 1]
```diff
@@ -298,20 +298,54 @@ When `storage_type=ha` (high availability), [Microsoft Azure NetApp Files](https
...
-| postgres_geo_redundant_backup_enabled | Whether to enable Geo-redundant for server backup. | bool | false | Not supported for the basic tier. |
-| postgres_administrator_login | The Administrator Login for the PostgreSQL server. Changing this forces a new resource to be created. | string | "pgadmin" | The admin login name cannot be azure_superuser, azure_pg_admin, admin, administrator, root, guest, or public. It cannot start with pg_. See: [Microsoft Quickstart Server Database](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-portal) |
-| postgres_administrator_password | The password associated with the postgres_administrator_login for the PostgreSQL Server. | string | | The password must contain between 8 and 128 characters and must contain characters from three of the following categories: English uppercase letters, English lowercase letters, numbers (0 through 9), and non-alphanumeric characters (!, $, #, %, etc.). |
-| postgres_server_version | The version of the Azure Database for PostgreSQL server instance. Valid values are "9.5", "9.6", "10.0", and "11". Changing this forces a new resource to be created.| string | "11" | |
```
Comment (hahewlet) [inline docs/CONFIG-VARS.md:312]: Can we remove the references to any version below 11? Viya doesn't support anything but 11 or 12 and azure doesn't support v12 of postgres yet.

File: docs/CONFIG-VARS.md [hunk 2]
```diff
@@ -298,20 +298,54 @@ When `storage_type=ha` (high availability), [Microsoft Azure NetApp Files](https
...
+| geo_redundant_backup_enabled | Enable Geo-redundant or not for server backup | bool | false | Not supported for the basic tier. |
+| administrator_login | The Administrator Login for the PostgreSQL Server. Changing this forces a new resource to be created. | string | "pgadmin" | The admin login name cannot be azure_superuser, azure_pg_admin, admin, administrator, root, guest, or public. It cannot start with pg_. See: [Microsoft Quickstart Server Database](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-portal) |
+| administrator_password | The Password associated with the administrator_login for the PostgreSQL Server | string | "my$up3rS3cretPassw0rd" | The password must contain between 8 and 128 characters and must contain characters from three of the following categories: English uppercase letters, English lowercase letters, numbers (0 through 9), and non-alphanumeric characters (!, $, #, %, etc.). |
+| server_version | The version of the Azure Database for PostgreSQL server instance. Valid values are "9.5", "9.6", "10.0", and "11". Changing this forces a new resource to be created.| string | "11" | |
```
Comment (hahewlet) [inline docs/CONFIG-VARS.md:328]: Can we remove the references to any version below 11? Viya doesn't support anything but 11 or 12 and azure doesn't support v12 of postgres yet.

File: docs/CONFIG-VARS.md [hunk 3]
```diff
@@ -298,20 +298,54 @@ When `storage_type=ha` (high availability), [Microsoft Azure NetApp Files](https
...
+| ssl_enforcement_enabled | Enforce SSL on connection to the Azure Database for PostgreSQL server instance | bool | true | |
+| configurations | Configurations to enable on the PostgreSQL Database server instance | map | {} | |
```
Comment (hahewlet) [inline docs/CONFIG-VARS.md:330]: Can we have a link to further doc on configurations? The link for a similar value, database_flags, in viya4-iac-gcp was very helpful.

Comment (thpang) [reply docs/CONFIG-VARS.md]: Done

**Review by thpang — COMMENTED** [review]
Added info requested.

**Comment by hahewlet** [comment]
Testing is complete. signing off

**Pull request merged**